man/bst.1.scd

```diff
@@ -229,6 +229,42 @@ _VAR=value_ before the executable to run.
 
 	Supported options are described in more detail in the *NETWORKING* section.
 
+\--cgroup <path>
+	Specify the cgroup directory that bst will join or operate within. 
+
+	If no climits are provided, then bst will join the cgroup at the provided <path>.
+	If climits are provided then bst will create cgroup sub-hierarchies at the <path>.
```

Comment:
I think it might be more accurate to say that bst will always join the specified cgroup, and that it will create subhierarchies as necessary.